demo/tests/test_end_point.py

```diff
@@ -889,6 +889,38 @@ def test_create_form_conditions(self):
         serializer = FormidableSerializer(data=data)
         self.assertTrue(serializer.is_valid(), serializer.errors)
         form = serializer.save()
+        form = Formidable.objects.get(pk=form.pk)
```

Comment:
you can use ``refresh_from_db()`` https://docs.djangoproject.com/en/1.11/ref/models/instances/#django.db.models.Model.refresh_from_db